sway/commands/title_format.c

```diff
@@ -11,16 +12,20 @@ struct cmd_results *cmd_title_format(int argc, char **argv) {
 		return error;
 	}
 	struct sway_container *container = config->handler_context.container;
-	if (!container || !container->view) {
+	if (!container) {
 		return cmd_results_new(CMD_INVALID,
-				"Only views can have a title_format");
+						 "Only valid containers can have a title_format");
 	}
-	struct sway_view *view = container->view;
 	char *format = join_args(argv, argc);
-	if (view->title_format) {
-		free(view->title_format);
+	if (container->title_format) {
+		free(container->title_format);
+	}
+	container->title_format = format;
+	if (container->view) {
+		view_update_title(container->view, true);
+	} else {
+		container_update_representation(container);
+		container_update_title_bar(container);
```

Comment:
I don't think this call is needed. I think `container_update_representation` will update the title bar itself.

Reply:
Yeah, it does, thank you.